PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -189,7 +189,7 @@ def miniAOD_customizeCommon(process):
 	 valueLabels = cms.vstring('pt','emEnergyFraction'),
 	 lazyParser = cms.bool(True) )
     task.add(process.caloJetMap)
-    process.patJets.userData.userFloats.src += [ cms.InputTag("caloJetMap:pt"), cms.InputTag("caloJetMap:emEnergyFraction") ]
+#    process.patJets.userData.userFloats.src += [ cms.InputTag("caloJetMap:pt"), cms.InputTag("caloJetMap:emEnergyFraction") ]
```

Comment:
It looks like my comment from a week ago still applies:https://github.com/cms-sw/cmssw/pull/19446/files#r124269867
Based on the discussion, it looks like it is OK to drop the userFloats.

So, the commented out line should be removed or there should be a clear comment added why it is still needed